src/translator/helper.ts

```diff
@@ -52,3 +64,66 @@ export const createConditionalRenderingObject = (
 
     return Object.keys(conditionObj).length === 0 ? [] : [conditionObj];
 };
+
+export const translateConditionalRenderingObject = (
+    conditions: ISchemaCondition[],
```

Comment:
where possible, use the typings from FEE since it is meant to parse from it
```suggestion
    conditions: IYupConditionalValidationRule[],
```